logger.go

```diff
@@ -129,3 +158,38 @@ func (l *MicroLogger) processStack(keyVals []interface{}) []interface{} {
 
 	return keyVals
 }
+
+func (l *CtxMicroLogger) Debug(format string, params ...interface{}) {
```

Comment:
I was coming from a non-operator simple exporter service and there are no context instances in all cases, but now that you bring it up, it could be justified because most of the time there is one so majority kind of wins for convenience. I did cringe a bit with the intermediate struct instance to make `WithContext()` possible so that would fix it as well. I'll most likely change it to your suggestion.

Reply:
I'd say we should always use the context. That's our way to pass key/values now. If we have some legacy code then `context.TODO()` is fine.

Also small thing. I would add `f` at the end of the method names to denote that this will do string formatting.